web/marketing/css/style.css

```diff
@@ -60,12 +60,13 @@ hr{
     font-style: normal;
 }
 
-.data-row{
+.data-row,
+.data-row .col a{
     font-weight: normal;
     font-size: 14px;
     line-height: 49px;
     letter-spacing: -0.100741px;
-    color: #656565;
+    color: #656565 !important;
```

Comment:
Why `!important`? This seems like ordering in css is messed up.

Reply:
removed